tests/test_sharding/test_runner_cli.py

```
@@ -0,0 +1,679 @@
+        "run",
+        suite,
+        "--shard-count",
+        "2",
+        "--shard-index",
+        "0",
+        "--timeout-policy",
+        "skip",
+        "--deadline-seconds",
+        "0",
+    )
+    finalized = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)